src/components/cards/Card.js

```diff
@@ -31,14 +32,19 @@ export default function Card({
 
   const [msg, setMsg] = useState("");
   const [isCommentsVisible, setisCommentsVisible] = useState(false);
-  const [comment, setComment] = useState("");
+  const [commentLength, setCommentLength] = useState("");
+  const [commentChk, setCommentChk] = useState([]);
```

Comment:
`commentChk` is short for commentCheck? This would be my guess. If not you might think of renaming the state?